app/converters/gbfs_share_birrer_adjust_bicycle.py

```diff
@@ -0,0 +1,33 @@
+from typing import Union
+
+from app.base_converter import BaseConverter
+
+BICYCLE_ELECTRIC_MAX_RANGE_METERS = 60000
+
+
+class GbfsShareBirrerAdjustBicycleConverter(BaseConverter):
+    """
+    share_birrer_ch uses invalid form_factor 'bike'
+    """
+
+    hostnames = ['www.share-birrer.ch', 'auto-birrer.ch']
+
+    def convert(self, data: Union[dict, list], path: str) -> Union[dict, list]:
+        if not isinstance(data, dict):
+            return data
+
+        if path.endswith('/vehicle_types'):
+            fields = data.get('data')
+            if not isinstance(fields, dict):
+                return data
+            vehicle_types = fields.get('vehicle_types')
+            if not isinstance(vehicle_types, list):
+                return data
+            for vehicle_type in vehicle_types:
+                if vehicle_type.get('form_factor') == 'bike':
+                    vehicle_type['form_factor'] = 'bicycle'
+                    vehicle_type['propulsion_type'] = 'electric'
+                    vehicle_type['max_range_meters'] = BICYCLE_ELECTRIC_MAX_RANGE_METERS
```

Comment:
Besides this, birrer uses an invalid propulsion_type `hydrogen`:
<img width="314" height="202" alt="image" src="https://github.com/user-attachments/assets/f6634ccc-82b4-462f-ad78-4db4b75ac9c4" />


According to [v2.3 spec](https://github.com/MobilityData/gbfs/blob/v2.3/gbfs.md#vehicle_typesjson) this should be `hydrogen_fuel_cell `
I suggest to fix this here as well.